.devcontainer/development/Dockerfile.orig

```
@@ -0,0 +1,190 @@
+
+ARG TOMCAT_PATH=/usr/local/tomcat
+ARG DOCS_PATH='/home/oscar/development/volumes'
+ARG OSCAR_DOCUMENT="${DOCS_PATH}/OscarDocument"
+ARG DB_DOCS="/db-data/documents"
+ARG DOCS_DEST="/var/lib/OscarDocument/oscar/document"
+
+WORKDIR /workspace
+
+# Install required packages including Playwright browser dependencies and clean up apt lists
+# Note: Ubuntu 24.04 uses t64 packages (time_t 64-bit migration)
+RUN apt-get update && apt-get install -y --no-install-recommends \
```

Hadolint (2.14.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)